integration_tests/tests/v2Manage/future_manager/beds.cy.ts

```diff
@@ -27,25 +27,25 @@ context('Beds', () => {
     // Given I am signed in as a workflow manager
     signIn(['future_manager'])
 
-    // When I visit the rooms page
-    const bedsPage = V2BedsListPage.visit(premisesId)
+    // When I visit the V2 beds page
+    const v2BedsPage = V2BedsListPage.visit(premisesId)
 
-    // Then I should see all of the rooms listed
-    bedsPage.shouldShowBeds(bedSummaries, premisesId)
+    // Then I should see all of the beds listed
+    v2BedsPage.shouldShowBeds(bedSummaries, premisesId)
 
     // When I click on a bed
-    bedsPage.clickBed(bedDetail)
+    v2BedsPage.clickBed(bedDetail)
 
     // Then I should be taken to the bed page
     Page.verifyOnPage(V2BedShowPage, bedDetail.name)
 
-    // Give I'm on the bed page
-    const bedPage = V2BedShowPage.visit(premisesId, bedDetail)
+    // Give I'm on the V2 bed page
```

Comment:
Whilst we're here this should be `Given I'm on the V2 bed page`

Reply:
thks, well spotted! I've fixed that up.